redis/src/cluster_async/mod.rs

```diff
@@ -282,38 +287,42 @@ where
                 }
                 request.retry = request.retry.saturating_add(1);
 
-                if let Some(error_code) = err.code() {
-                    if error_code == "MOVED" || error_code == "ASK" {
-                        // Refresh slots and request again.
-                        request.info.excludes.clear();
-                        return Next::Err {
-                            request: this.request.take().unwrap(),
-                            error: err,
-                        }
-                        .into();
-                    } else if error_code == "TRYAGAIN" || error_code == "CLUSTERDOWN" {
+                match err.kind() {
+                    // TODO: Proper Ask implementation
```

Comment:
please add what's missing from the current implementation

Reply:
I'm working on a followup PR that should add the implementation shortly, so I think this is ok for now.